src/cluster/index.js

```diff
@@ -159,6 +159,24 @@ module.exports = class Cluster {
     }
   }
 
+  /**
```

Comment:
Make `addTargetTopic` delegate to `addMultipleTargetTopics`, since the implementation is exactly the same.